big-red-shuttle/StopsViewController.swift

```diff
@@ -61,7 +64,33 @@ class StopsViewController: UIViewController, GMSMapViewDelegate {
             marker.map = mapView
         }
     }
+    
+    
+    //drawing
+    func initPolyline() {
+        let stops = getStops()
+        let stopsA = Array(stops[1..<9]) //can only have a max of 8 waypoints (i.e. stops that don't include start and end)
```

Comment:
Will this always be equal to the number of stops? If so, we should base this range on the length of the array where we store the different stops